### PR TITLE
Add Proof of Address improvement fields

### PR DESCRIPTION
**Pull request by subenson**
## Details

https://github.com/Rebilly/backend/pull/10200

### CODE REVIEW

File: openapi/components/schemas/KycDocument/ProofOfAddress.yaml [hunk 1]
```diff
@@ -56,6 +56,23 @@ allOf:
                 description: 'The customer postal code if it was matched, null otherwise'
                 type: string
                 example: WC2N 5NF
+              wordCount:
+                description: 'The number of words in the document'
```
Comment (vykhrystiuk) [inline openapi/components/schemas/KycDocument/ProofOfAddress.yaml:60]: Descriptions require periods. (See: https://rebilly.slack.com/archives/C7U0AHYJ0/p1601565453042200)

File: openapi/components/schemas/KycDocument/ProofOfAddress.yaml [hunk 2]
```diff
@@ -56,6 +56,23 @@ allOf:
...
+                type: integer
+                example: 350
+              uniqueWords:
+                description: 'The number of unique words in the document'
```
Comment (ashkarpetin) [inline openapi/components/schemas/KycDocument/ProofOfAddress.yaml:64]: I missed that before, why single quote here?

Comment (svensono) [reply openapi/components/schemas/KycDocument/ProofOfAddress.yaml]: This is unintentional. I removed them.

File: openapi/components/schemas/KycDocument/ProofOfAddress.yaml (hunk 1 above)
Comment (svensono) [reply openapi/components/schemas/KycDocument/ProofOfAddress.yaml]: :+1:

**Review by ashkarpetin — DISMISSED** [review]

**Review by iamdevice — DISMISSED** [review]

**Review by vykhrystiuk — DISMISSED** [review]

**Comment by svensono** [comment]
I added `wordCountResult` and uniqueWordsResult` after approval.

**Review by removed-account-123 — DISMISSED** [review]

File: openapi/components/schemas/KycDocument/ProofOfAddress.yaml [hunk 3]
```diff
@@ -33,29 +33,54 @@ allOf:
...
-                description: 'The customer postal code if it was matched, null otherwise'
+                description: The customer postal code if it was matched, null otherwise.
                 type: string
                 example: WC2N 5NF
+              wordCount:
+                description: The number of words in the document.
+                type: integer
+                example: 350
+              wordCountResult:
+                description: Flag that indicates if the word count passed the threshold
```
Comment (vykhrystiuk) [inline openapi/components/schemas/KycDocument/ProofOfAddress.yaml:64]:
```suggestion
                description: Flag that indicates if the word count passed the threshold.
```

File: openapi/components/schemas/KycDocument/ProofOfAddress.yaml [hunk 4]
```diff
@@ -33,29 +33,54 @@ allOf:
...
+                type: boolean
+                example: true
+              uniqueWords:
+                description: The number of unique words in the document.
+                type: integer
+                example: 175
+              uniqueWordsResult:
+                description: Flag that indicates if the unique words passed the threshold
```
Comment (vykhrystiuk) [inline openapi/components/schemas/KycDocument/ProofOfAddress.yaml:72]:
```suggestion
                description: Flag that indicates if the unique words passed the threshold.
```

**Review by vykhrystiuk — APPROVED** [review]

**Review by removed-account-123 — APPROVED** [review]

**Pull request merged**